scalatest/src/main/scala/org/scalatest/tools/Framework.scala

```diff
@@ -708,7 +710,7 @@ class Framework extends SbtFramework {
           tracker,
           tagsToInclude,
           tagsToExclude,
-          td.selectors ++ autoSelectors,
+          (td.selectors ++ autoSelectors).map(_.nn),
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.